.github/workflows/build.yml

```diff
@@ -31,6 +32,7 @@ jobs:
 
       - name: Run unit test
         run: |
+          mkdir /var/tmp/worker0 /var/tmp/worker1 /var/tmp/worker2 /var/tmp/worker3
```

Comment:
Why only 4 paths? Do we have to manually edit this if we are running with different worker counts?

Reply:
In unit tests, it spawns workers only up to 4 workers. 
Yes, we need to edit this if the unit tests are written to spawn more than 4 workers. 
This change is required due to some recent updates in the GitHub workflows / Docker version. Previously, the folders related to the workers were created automatically when the mount path was defined. (This is still the case in my local setup as well as in GKE).

Reply:
Ok, since it is a unit test we could proceed as it is.